src/data/multiset/fold.lean

```diff
@@ -80,6 +81,31 @@ end
 
 end fold
 
+section order
+
+@[to_additive]
+lemma prod_le_of_forall_le {α : Type*} [ordered_comm_monoid α]
+  (l : multiset α) (n : α) (h : ∀ (x ∈ l), x ≤ n) :
+  l.prod ≤ n ^ l.card :=
+begin
+  induction l using quotient.induction_on,
+  simpa using list.prod_le_of_forall_le _ _ h
+end
```

Comment:
Similarly this lemma is not about `fold`, so probably belongs earlier in the `multiset` import tree